### ISSUE TITLE
mdbg missing

### ISSUE THREAD

**Issue by casterle** [dotnet/docs#10220]
I've got VS 2017 installed and can find no trace of mdgb.exe.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 60439c92-7612-ef8a-c00a-1a1869300d5e
* Version Independent ID: 8b6d240f-aca1-646b-d0a5-b9a783e2c3da
* Content: [.NET Framework Tools](https://docs.microsoft.com/en-us/dotnet/framework/tools/)
* Content Source: [docs/framework/tools/index.md](https://github.com/dotnet/docs/blob/master/docs/framework/tools/index.md)
* Product: **dotnet-framework**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

**Comment by mairaw** [comment]
Thanks for your feedback @casterle.
 
The MDBG topic covers where you can download the tool from:
https://docs.microsoft.com/en-us/dotnet/framework/tools/mdbg-exe

**Comment by casterle** [comment]
Thanks, I finally found that. 

 

The problem I was complaining about was the implication that installing VS and using the VS command prompt, I could run the utilities listed below. 

 

I didn’t follow the link to the mdgb page because I already had instructions for using it - which implied that it should have been installed already – and I was searching for it when I found the page I was commenting on. 

 

Adding a note *on that page*
